index.html

```diff
@@ -66,33 +78,40 @@ <h2 class="Description1">I'am a Software Developer</h2>
             ><img src="images/Icon - Twitter.png" alt="Twitter icon"
           /></a>
         </li>
+        <li class="medium">
+          <a href=""
+            ><img src="images/medium.svg" alt="Medium icon"
+          /></a>
+        </li>
       </ul>
     </section>
     <section class="grid-container">
       <div class="main-div">
         <div class="top-div">
           <h2 id="MyRecentWorks">My Recent Works</h2>
           <img src="images/Vector.png" alt="horizontal-line" id="hr-line" />
+        </div>
+        <div class="butm-div">
           <img
             src="images/Img Placeholder.png"
             alt="yoga image"
             id="yoga-image"
           />
-        </div>
-        <div class="butm-div">
-          <h2>Multi-Post Stories</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="list1">
-            <li>CSS</li>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <a href="#" class="button">See project</a>
+          <div class="project">
+            <h2>Multi-Post Stories</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="list1">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a href="#" class="button">See project</a>
+          </div>
         </div>
```

Comment:
- [ ] You did a great job 😃 on the project, but I see that your cards portfolio it is not been ordered on the page, so please try to organize the cards in the section to make it nice like the Figma 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/230614069-905e99ad-8dfc-4f7e-964d-526d37bf1bc3.png)  |   ![Recording 2023-04-07 at 15 05 44](https://user-images.githubusercontent.com/83360636/230613891-817877e4-48df-4116-bcf1-2e984fb232db.gif)  |